src/Components/Web/src/Forms/InputCheckbox.cs

```diff
@@ -21,6 +21,12 @@ namespace Microsoft.AspNetCore.Components.Forms
     /// </summary>
     public class InputCheckbox : InputBase<bool>
     {
+        /// <summary>
+        /// Gets or sets the associated <see cref="ElementReference"/>
+        /// <para>This value is used to gain access to helper methods on the input elment.</para>
+        /// </summary>
+        public ElementReference? InputElement { get; protected set; }
```

Comment:
```suggestion
        /// <summary>
        /// Gets or sets the associated <see cref="ElementReference"/>.
        /// </summary>
        public ElementReference? InputElement { get; protected set; }
```

I think this is enough - the uses of this property aren't limited to accessing helper methods (you could pass the `ElementReference` to JS, etc.).

Same applies to the other files.

Reply:
Agree